docs/specs/aerogear-offline/index.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+section: docs
+title: AeroGear Offline API proposal
+---
+
+<span class="label label-warning">Status: Experimental</span>
+
+# Goals
+
+1. To manage resources (assumed files) on the mobile devices which are retrieved from transitory sources (ie online, SD cards, etc).
+2. To provide a way to secure, and expire offline resource. (ie encryption and TTL policies)
+
+# API
+
+## Implementation 
+
+### Storage API
+ * `get` : *URL* **url**, *Callback* **callback**   
+This should check and see if a file related to the URL is available. 
+     * If the file is locally stored, the file should be passed to the callback's success method. 
+     * If the file is not it should be fetched, saved, and returned to the callback's success method.
```

Comment:
URL's aren't necessarily HTTP ;) put yes there will be a goal to add metadata to manage a file expiring.